test/Microsoft.ML.TestFramework/BaseTestClass.cs

```diff
@@ -66,33 +85,34 @@ protected virtual void Cleanup()
         {
         }
 
-        protected string RootDir => _rootDir;
-        protected string OutDir => _outDir;
+        protected static string RootDir { get; }
```

Comment:
I would name this `RepoRoot` or `RepoRootDir`. Just "RootDir" isn't very descriptive.